packages/services/serverless.yml

```diff
@@ -21,12 +21,12 @@ package:
     - .md
   include:
     - ./*-function.json
-    
+
 functions:
   sync-service:
     handler: dist/functions/azure/sync-service.run
     events:
-      - serviceBus: 
+      - serviceBus:
         x-azure-settings:
           queueName: webhint-results
           name: message
```

Comment:
Don't worry about changes in this file for now. To make Serverless.yml work, step 1 was to convert everything to functions, which we are close to completing. Step to is for me to either send a PR to them, or fork our own version and update all the Azure dependencies and some logic because they are lagging behind by more than a year.